src/low-preview/ci.service.js

```diff
@@ -213,4 +213,40 @@ export default class CIResponsive {
       setBackgroundSrc(imgNode, cloudimageUrl, lazy, src, isSVG, dataSrcAttr);
     }
   }
+
+  setSrc(src, element) {
+    const {imgSelector, bgSelector} = this.config;
+    const isProcessed = element.classList.contains('ci-image');
+    const isImage = element.hasAttribute(imgSelector);
+    const isBackground = element.hasAttribute(bgSelector);
+
+    if (isImage) {
+
+      if (isProcessed) {
+        const elementParent = element.parentNode;
+        element.classList.remove('ci-image-loaded');
+        element.classList.remove('lazyloaded');
+        element.classList.remove('ci-image');
+
+        element.setAttribute(imgSelector, src);
+        elementParent.parentNode.replaceChild(element, elementParent);
+        this.getBasicInfo(element, false, false, 'image');
+      }
+
+      if (!isProcessed) {
+        this.getBasicInfo(element, false, false, 'image');
+      }
+    }
+
+    if (isBackground) {
```

Comment:
for background we need also to check if it's processed
by checking the first child classlist includes ci-image-loaded